lib/doekbase/data_api/annotation/genome_annotation/api.py

```diff
@@ -440,15 +469,24 @@ def get_feature_types(self):
         return feature_types
 
     def get_feature_ids(self, filters=None, group_by="type"):
-        # no choice but to pull all features
-        features = self._get_features()
+        if filters is None:
+            filters = {}
+
+        valid_filters = ["type_list", "region_list", "function_list", "alias_list"]
```

Comment:
these should be class constants with a "#:" style docstring/comment, so they show up in the docs and can be referred to within docstrings from the other methods.